plugins/advisors/osv/src/funTest/kotlin/OsvFunTest.kt

```diff
@@ -19,7 +19,7 @@
 
 package org.ossreviewtoolkit.plugins.advisors.osv
 
-import io.kotest.core.spec.style.StringSpec
+import io.kotest.core.spec.style.WordSpec
```

Comment:
Please remember to provide proper rationales in commit messages. I guess here the motivation was to avoid some wording repetition, but it comes at the cost of increasing indentation.

As `retrievePackageFindings()` in this case is the only function being tested, this could maybe also have been solved by giving the tests names that do not mention the function name at all.